.travis.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       env:
         - CLANG_SANITIZER=TSAN
         - *common-job-env
-    - if: false
+    - if: type != pull_request
```

Comment:
btw: maybe this should be restricted to "master"?  (i.e. to not run on release branches; or if useful there then likely needs to use another channel (or something alike))

Reply:
well, the plan is to also push to release channel. In theory, that's decided/controlled by https://github.com/neovim/neovim/blob/5c9063cf2d115b652ccdde6f72afc464b7085a18/ci/snap/deploy.sh . 

Not clear if it works yet.